governance.md

```diff
@@ -1,27 +1,30 @@
 ---
 layout: contribute
-title: Governance
+title: Contribution policy
 ---
 
-# Governance
+# Contribution policy
 
-The Bazel project is led by a core group of contributors, initially Googlers, and managed by the
-community. The group of core contributors is self-managing - core contributors are added by two
+This page covers Bazel's governance model and contribution policy.
+
+## Governance model
+
+The Bazel project is led by a core group of contributors,  and managed by the
+community. At this time, all core contributors work for Google, but this will hopefully
+change over time. The group of core contributors is self-managing - core contributors are added by two
 supporting votes from core contributors on the mailing list and no veto within four business days.
 We expect that new contributors will submit a number of patches before they become core
 contributors.
 
-## Accepting Contributions
+Contact the core team at <a href="mailto:bazel-core@googlegroups.com">
+bazel-core@googlegroups.com</a>.
 
-Please also see our [contribution guidelines](contributing.html).
```

Comment:
I think it's helpful to have a link to the contribution guidelines. Perhaps after the list have it as a standalone paragraph that says "For more details on contributing to Bazel, see our [contribution guidelines](contributing.html)."

Reply:
Done.